### PR TITLE
Adding rtcJsVersion to softphone report

### PR DESCRIPTION
**Pull request by yiming-amzn**
*Issue #, if available:*
Adding rtcJsVersion to the softphone report so that we can track the rtcJs upgrade status on the server side.
*Description of changes:*
Added grunt-replace to read the version number from package.json and inject that value to the softphone report object as the default value.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Akash1684 — APPROVED** [review]

File: Gruntfile.js [hunk 1]
```diff
@@ -66,7 +66,24 @@ module.exports = function (grunt) {
...
             }
-        }
+        },
+        replace: {
+            dist: {
+              options: {
+                patterns: [
+                  {
+                    match: 'RTC_JS_VERSION',
+                    replacement: '<%= pkg.version %>'
```
Comment (Vignendra) [inline Gruntfile.js:76]: Wouldn't the change to increase the pkg.version also be part of this commit change.

File: Gruntfile.js [hunk 2]
```diff
@@ -66,7 +66,24 @@ module.exports = function (grunt) {
                     protocol: 'https'
                 }
             }
-        }
+        },
```
Comment (Vignendra) [inline Gruntfile.js:69]: As discussed offline make sure to squash all the commits into a single commit before merging.

File: Gruntfile.js (hunk 1 above)
Comment (yiming-amzn) [reply Gruntfile.js:76]: This is my understanding after reading the scripts in package.json. I’m going to merge my changes to master branch. then I switch to bumpVersion branch based on master branch, then increase version number, and build those release files, and create a pr to merge the version number change and new release files from bumpVersion branch to master branch (edited)

Comment (yiming-amzn) [reply Gruntfile.js:76]: the short answer is the pkg.version upgrade will be a different pr, along with the new 3 build files

File: Gruntfile.js (hunk 2 above)
Comment (yiming-amzn) [reply Gruntfile.js:69]: I disabled Allow merge commits (Add all commits from the head branch to the base branch with a merge commit.) and allowed "Allow squash merging "(Combine all commits from the head branch into a single commit in the base branch.) in https://github.com/aws/connect-rtc-js/settings to make sure I'm not pushing commits separately

**Review by Vignendra — APPROVED** [review]

**Pull request merged**